app/views/contexts/individual_response_context.py

```diff
@@ -0,0 +1,34 @@
+from typing import Mapping
+
+from app.jinja_filters import RadioConfig
+from app.views.contexts import Context
+from app.forms.questionnaire_form import QuestionnaireForm
+from app.views.contexts.question import build_question_context
+
+
+class IndividualResponseContext(Context):
+    @staticmethod
+    def get_context(content, form: QuestionnaireForm) -> Mapping:
+        legend = content["question"]["title"]
+        answer = content["question"]["answers"][0]
+
+        content = build_question_context(content, form)
+        content["contact_us_url"] = "https://census.gov.uk/contact-us"
```

Comment:
We need to map these as they different based on region/lang.
NISRA: `https://census.gov.uk/ni/contact-us/`
Lang `cy`: `https://cyfrifiad.gov.uk/cysylltu-a-ni/`
Else: `https://census.gov.uk/contact-us/`

Currently, these mapped in error page HTML files.